src/Fixture/AbstractAssetFixture.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Fixture;
+
+use Pimcore\Model\Asset;
+use Pimcore\Model\Asset\Image;
+
+abstract class AbstractAssetFixture extends AbstractFixture
```

Comment:
The class has been named  `AbstractAssetFixture`, but its `create` method creates an object of type `Asset\Image`.
Therefore I would call it: `ImageAssetFixture`.
There could also be alternatives like `VideoAssetFixture` and `DocumentAssetFixture` which would do more or less the same but with `Asset\Video` and `Asset\Document` results.

And of course there could an (abstract) implementation of `AbstractAssetFixture` which is really creating `Asset`s.